docs/howto/features/community-docs.md

```diff
@@ -0,0 +1,91 @@
+# Integrate community-documentation with Jupyter Book
+
+With [`#5045`][mvp], we now support the ability to connect community-documentation with a deployed Hub. From an infrastructure perspective, this is a thin integration in which we deliver the following features:
```

Comment:
I think I'd rather replace this alt text with something more descriptive than the PR number